versions/3.0.md

```diff
@@ -1128,20 +1127,38 @@ A request body with a referenced model definition.
       "schema": {
         "$ref": "#/components/schemas/User"
       },
-      "examples": [ "http://foo.bar/examples/user-example.json" ]
+      "examples": {
```

Comment:
It appears to be a free-form description/label/name but the intent is not explained in the _"Examples Object"_ description.